CHANGES_NEXT_RELEASE

```diff
@@ -15,6 +15,7 @@
 - Fix: missing error payload for NotFound error in 'PUT /v2/entities/<eId>/attrs/<attrName>' and 'PUT /v2/entities/<eId>/attrs/<attrName>/value' operations (Issue #1909)
 - Fix: NGSIv2 "InvalidModification" errors changed to "Unprocessable" (according to rules defined in #1286)
 - Fix: NGSIv2 geoquery syntax errors using 400 Bad Request instead of 422 Unprocessable Entity (according to rules defined in #1286)
+- Fix: NGSIv2 URI PATH must be all in lowercase, otherwise not found (Issue #2057)
```

Comment:
"not found" could be me bit missleading... (it could be missinterpreted as *entity* not found, *attribute* not found).

I'd suggest:

Fix: NGSIv2 URI PATH must be all in lowercase, otherwise "sevice not found" (Issue #2057)